pages/auth/index.tsx

```diff
@@ -0,0 +1,338 @@
+import React, {
+  useCallback,
+  useMemo,
+  useRef,
+  useState,
+  KeyboardEvent,
+  useEffect,
+} from "react";
+import Head from "next/head";
+import {
+  Box,
+  Container,
+  TextField,
+  Typography,
+  Button,
+  Dialog,
+  DialogTitle,
+  DialogActions,
+} from "@material-ui/core";
+import { gql, useMutation } from "@apollo/client";
+import { useRouter } from "next/router";
+import AuthLogo from "../../components/AuthLogo";
+
+const SIGNIN = gql`
```

Comment:
This constant should be named as the mutation(capitalized) like: `EMAIL_VALIDATION`